<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QMainWindow" name="SettingWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/setting.ico</normalon>
   </iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="termGroupBox">
      <property name="title">
       <string>This term</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>begins on</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="termStartDateEdit">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2019</year>
               <month>2</month>
               <day>25</day>
              </datetime>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>has</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="totalWeekBox">
             <property name="suffix">
              <string> weeks</string>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>15</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Time of sections</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <row>
          <property name="text">
           <string>1st</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2nd</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3rd</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4th</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5th</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6th</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7th</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8th</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9th</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>10th</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>11th</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Start time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>End time</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>8:50</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>9:35</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>9:40</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>10:25</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>10:40</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>11:25</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>11:30</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>12:15</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>14:00</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>14:45</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>14:50</string>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string>15:35</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>15:45</string>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string>16:30</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>16:35</string>
          </property>
         </item>
         <item row="7" column="1">
          <property name="text">
           <string>17:20</string>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>19:00</string>
          </property>
         </item>
         <item row="8" column="1">
          <property name="text">
           <string>19:45</string>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string>19:55</string>
          </property>
         </item>
         <item row="9" column="1">
          <property name="text">
           <string>20:40</string>
          </property>
         </item>
         <item row="10" column="0">
          <property name="text">
           <string>20:50</string>
          </property>
         </item>
         <item row="10" column="1">
          <property name="text">
           <string>21:35</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
